src/api/routes/rosetta/network.ts

```diff
@@ -0,0 +1,50 @@
+import * as express from 'express';
+import { addAsync, RouterWithAsync } from '@awaitjs/express';
+import { DataStore } from '../../../datastore/common';
+import {
+  RosettaConstants,
+  RosettaOperationTypes,
+  RosettaOperationStatuses,
+  RosettaErrors,
+} from '../../rosetta-constants';
+
+export function createRNetworkRouter(db: DataStore): RouterWithAsync {
+  const router = addAsync(express.Router());
+
+  router.post('/list', (_req, res) => {
+    const response = {
+      network_identifiers: [
+        {
+          blockchain: RosettaConstants.blockchain,
+          network: RosettaConstants.network,
+        },
+      ],
+    };
+
+    res.json(response);
+  });
+
+  router.post('/status', (req, res) => {
+    res.json({ status: 'ready' });
+  });
+
+  router.post('/options', (_req, res) => {
+    const response = {
+      version: {
+        rosetta_version: RosettaConstants.rosettaVersion,
+        node_version: process.version,
+        middleware_version: process.env.npm_package_version,
```

Comment:
I don't think this env var is available if the compiled js is ran via `node lib/index.js`. 
Maybe better to use the `require('./package.json').version` technique?